docs/xroad_security_server_toolkit_user_guide.md

```diff
@@ -96,7 +97,16 @@ The document is intended for readers with a good knowledge of Linux server manag
 The X-Road Security Server Toolkit package can be installed using PIP:
 
 ```
-$ pip install xrdsst --extra-index-url https://xroad-toolkit.s3-eu-west-1.amazonaws.com/xrdsst
+$ pip install --index-url http://xroad-toolkit.s3-website-eu-west-1.amazonaws.com/xrdsst xrdsst
+```
+
+The downloaded and signed packages can be verified with a public key:
+
+**The public key can be obtained from : https://pgp.mit.edu/ by typing ``jenkins@niis.org`` into the search string field.**
```

Comment:
Here it would be good to list key fingerprint explicitly, like in other X-Road package documentations. The import command then can also be given together with ``--with-fingerprint`` option, -- this search box at pgp.mit.edu has been verified to work ATM, but is not very graceful and it would be easily possible to add spam keys that also show up when searching for 'jenkins@niis.org'.

Is this key not available at SKS keyserver pools?

Reply:
This key has been added according to some instructions: 
Upload to sks keyserver pool
Select a keyserver from the pool that provides a webinterface, for instance https://pgp.mit.edu
Paste the content of the file "john_doe.pub" unmodified and including the ----BEGIN and ---END lines into the form
Click on "submit this key to the keyserver!"

So I do not see any issue with that.

I can add the fingerprint to UG.

Reply:
Added

Reply:
Now please just change import command to also verify fingerprint and this is good to merge and simple to use.

Reply:
Changed